substrate/frame/collective/src/migrations/v5.rs

```diff
@@ -0,0 +1,196 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::{Config, Pallet};
+use codec::{Decode, Encode};
+use frame_support::{
+	pallet_prelude::{OptionQuery, TypeInfo, ValueQuery},
+	sp_runtime::RuntimeDebug,
+	traits::{Get, OnRuntimeUpgrade, QueryPreimage, StorePreimage},
+	weights::Weight,
+	BoundedVec, Identity,
+};
+use frame_system::pallet_prelude::BlockNumberFor;
+use sp_std::prelude::*;
+
+pub mod old {
```

Comment:
At some point decided a better pattern is to avoid 'old' and explicitly mention what version this storage is representing 
```suggestion
pub mod v4 {
```